crates/uplc/src/lib.rs

```diff
@@ -6,3 +6,14 @@ pub mod machine;
 pub mod parser;
 mod pretty;
 pub mod program_builder;
+
+pub use pallas_primitives::alonzo::PlutusData;
+use pallas_primitives::Fragment;
+
+pub fn plutus_data(bytes: &[u8]) -> PlutusData {
```

Comment:
can we make this `pub(crate)`? do we need this exposed to users of the library? If we do want to expose this we should return a `Result`, so that the user can decide themselves if they want to unwrap.